lib/event_framework/event_store/sink.rb

```diff
@@ -3,6 +3,7 @@ module EventStore
     class Sink
       AggregateIdMismatchError = Class.new(Error)
       UnableToGetLockError = Class.new(Error)
+      MultipleAggregateIDs = Class.new(Error)
```

Comment:
```suggestion
      MultipleAggregateIDsError = Class.new(Error)
```

Should we make this consistent with the other error class names?

Reply:
Good call. Fixed.